fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -879,8 +879,9 @@ ACTOR Future<Void> applyMetadataToCommittedTransactions(CommitBatchContext* self
 // Message the sequencer to obtain the previous commit version for each storage server's tag
```

Comment:
We need to obtain the previous commit version for each tlog to which we are going to send the commit version/ mutations to?

Reply:
// Message the sequencer to obtain the previous commit version for each tlog to which we are going to send 
// the commit version/ mutations to